modulefiles/module_base.orion

```diff
@@ -1,52 +1,41 @@
-#%Module######################################################################
-##
-##      FV3GFS prerequisites
-##
+#%Module################################################
+# FV3GFS prerequisites
+########################################################
 
 module use /apps/contrib/NCEP/libs/hpc-stack/modulefiles/stack
 module load hpc/1.1.0
 module load hpc-intel/2018.4
-module load hpc-impi/2018.4
-
-#module load hpss/hpss
-module load nco/4.8.1
-module load gempak/7.5.1
 
-module load grib_util/1.2.2
-module load prod_util/1.2.2
+# gsi
+module unload mkl/2020.2
+module load mkl/2018.4
 
-module load crtm/2.3.0
-setenv CRTM_FIX /apps/contrib/NCEPLIBS/orion/fix/crtm_v2.3.0
+module load hpc-impi/2018.4
+module load slurm/19.05.3-2
 
-module load jasper/2.0.25
-module load zlib/1.2.11
-module load png/1.6.35
+# waveprep
+module load cdo/1.9.5
 
-module load hdf5/1.10.6
-module load netcdf/4.7.4
 module load pio/2.5.2
 module load esmf/8_1_1
 module load fms/2020.04.03
-
+module load grib_util/1.2.2
+module load prod_util/1.2.2
+module load gempak/7.5.1
 module load bacio/2.4.1
-module load g2/3.4.1
-module load g2tmpl/1.9.1
+module load upp/10.0.6
```

Comment:
UPP module requires several of the libraries loaded.
Please don't move the module loads around unless you have a sound reason to do so.

Reply:
load upp module also load g2 and g2tmpl automatically. That is the reason for the change.

Reply:
Answered in Files changed page.

Reply:
In a global-workflow PR's meeting on 12/8 3:45p, Rahul requested to keep module load upp in the same location, keep module load g2 and g2tmpl in the same location, and update the version number for g2 and g2tmpl to match upp.